packages/shared/src/schemas.ts

```diff
@@ -83,7 +83,7 @@ export const openSwapDepositChannelSchema = z
     destChain: chain,
     destAddress: z.string(),
     amount: numericString,
-    ccmMetadata: ccmMetadataSchema.optional(),
+    ccmParams: ccmParamsSchema.optional(),
```

Comment:
should keep them both for now to ensure we are properly validating the args
```suggestion
    ccmParams: ccmParamsSchema.optional(),
    ccmMetadata: ccmMetadataSchema.optional(), // deprecated
```

Reply:
makes sense! but i think ill omit the `ccmMetadata` from the type to prevent people from using the deprecated one. too bad that txpescript does not really have a way to deprecate properties

Reply:
laright this type isnt exposed anyway

Reply:
okay should show up as deprecated on the types now